platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -671,11 +678,27 @@ check_package_manager() {
 }
 
 check_ca_cert() {
```

Comment:
Can you also ensure that check_ca_cert is shown as skipped if we run the package manager check ( and it passes?) . That way we always have a consistent number of checks we run